book/website/ethical-research/ethics-open-source-governance.md

```diff
@@ -0,0 +1,99 @@
+(er-ethics-open-source-governance)=
+# Ethical Considerations when Choosing an Open Source Governance Model
+
+(er-ethics-open-source-governance-summary)=
+## Summary
+
+Open source projects are rooted in ethical principles. 
+Sharing the code so that anyone can use it, welcoming contributors and creating communities around specific projects are the core of open source. 
+But open source communities also have the responsibility to ensure they are a safe, inclusive environment, understand who might be affected by their projects or check that their tools are not being misused. 
+Therefore, Open Source Governance Models should embed those ethical considerations within their structure, but the framework to do so is far from well-established. 
```

Comment:
Maybe define what Open Source Governance models means here - and/or provide reference.